### PR TITLE
TELCODOCS-260/262  - OCP 4.9 release note

### PR DESCRIPTION
**Pull request by aireilly**
enterprise-4.9 release note for PTP fast event notifications for the following doc epics:

https://issues.redhat.com/browse/TELCODOCS-260
https://issues.redhat.com/browse/TELCODOCS-262 

Preview:

https://deploy-preview-36427--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-ptp-fast-event-notifications

### CODE REVIEW

**Review by aneeshkp — APPROVED** [review]

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -113,6 +113,11 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
 [id="ocp-4-9-networking"]
 === Networking
 
+[id="ocp-4-9-ptp-fast-event-notifications"]
+==== Monitoring PTP fast events with the PTP fast events notifications framework
```
Comment (ogradyp) [inline release_notes/ocp-4-9-release-notes.adoc:117]: I think the second occurrence of 'events' here should be singular. 
So:
...with the PTP fast event notifications framework

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -113,6 +113,11 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
...
+
+Fast event notifications for PTP events are now available for bare-metal clusters. The PTP Operator generates event notifications for every configured PTP capable network interface. Events are made available through a REST API for applications running on the same node. Fast event notifications are transported by an AMQP message bus provided by the AMQ Interconnect Operator. For more information, see xref:../networking/configuring-ptp.adoc#cnf-about-ptp-and-clock-synchronization_configuring-ptp[About PTP and clock synchronization error events].
```
Comment (ogradyp) [inline release_notes/ocp-4-9-release-notes.adoc:119]: I think 'PTP capable' should be hyphenated.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -113,6 +113,11 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
 [id="ocp-4-9-networking"]
 === Networking
 
+[id="ocp-4-9-ptp-fast-event-notifications"]
+==== Monitoring PTP fast events with the PTP fast event notification framework
+
+Fast event notifications for PTP events are now available for bare-metal clusters. The PTP Operator generates event notifications for every configured PTP-capable network interface. Events are made available through a REST API for applications running on the same node. Fast event notifications are transported by an Advanced Message Queuing Protocol (AMQP) message bus provided by the AMQ Interconnect Operator. For more information, see xref:../networking/configuring-ptp.adoc#cnf-about-ptp-and-clock-synchronization_configuring-ptp[About PTP and clock synchronization error events].
```
Comment (kalexand-rh) [inline release_notes/ocp-4-9-release-notes.adoc:119]: Telco docs appear to be the only place to use "bare-metal" instead of "bare metal."

**Review by kalexand-rh — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-9-release-notes.adoc (hunk 3 above)
Comment (aireilly) [reply release_notes/ocp-4-9-release-notes.adoc:119]: "Bare-metal" is the adjectival form, "bare metal" is the noun form. https://redhat-documentation.github.io/supplementary-style-guide/#baremetal

https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal

**Pull request merged**